### PR TITLE
Integrate user pages

### PR DESCRIPTION
**Pull request by Gustav-Simonsson**

### CODE REVIEW

**Comment by Gustav-Simonsson** [comment]
@jesuscript Not sure if this approach is a good one - please review. Also user profile edit modal is still not integrated, due to questions around field validations.

**Comment by Gustav-Simonsson** [comment]
Also, layout is a bit funky in this approach :) it needs some polish - e.g. avatar pic and points are outside the grey box.

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,7 @@ packages/
 config/*
 todo.org
 .DS_Store
-build/
\ No newline at end of file
+build/
+
+*#
+*~
```
Comment (jesuscript) [inline .gitignore]:
```
(setq backup-directory-alist '(("." . "~/.emacs.d/backup"))
      backup-by-copying t    ; Don't delink hardlinks                                                                                                                                                       
      version-control t      ; Use version numbers on backups                                                                                                                                               
      delete-old-versions t  ; Automatically delete excess backups                                                                                                                                          
      kept-new-versions 20   ; how many of the newest versions to keep                                                                                                                                      
      kept-old-versions 5    ; and how many of the old                                                                                                                                                      
      )
```

File: client/templates/profile.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Template.profile.helpers({
+  isUserProfile: function(profileid){
+  	var user = Meteor.user();
```
Comment (jesuscript) [inline client/templates/profile.js]: hmmmmmmm. Spaces and tabs you mix. In a dark place you will find yourself. hmmm.

**Pull request merged**